eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Ratify.hs

```diff
@@ -84,32 +85,27 @@ instance
 
 --- Constants
 
-ccThreshold :: Int
-ccThreshold = 0
+-- ccThreshold :: Int
+-- ccThreshold = 0
```

Comment:
The  ccThreshold check removal is also temporary, no? Maybe it would be good to add a comment explaining that this is why this is commented out.  We could also leave that check commented we a TODO so we don't forget about it?

Reply:
We won't forget about this, because those hard coded thresholds themselves are temporary. These thresholds will be coming from PParams in the future.